hashes/src/util.rs

```diff
@@ -110,28 +110,77 @@ macro_rules! engine_input_impl(
 
 
 /// Creates a new newtype around a [`Hash`] type.
+///
+/// The syntax is similar to the usual tuple struct syntax:
+///
+/// ```
+/// # use bitcoin_hashes::{hash_newtype, sha256};
+/// hash_newtype! {
+///     /// Hash of `Foo`.
+///     pub struct MyNewtype(pub sha256::Hash);
+/// }
+/// ```
+///
+/// You can use any valid visibility specifier in place of `pub` or you can leave it out if you
+/// don't want the type or its field to be private.
+///
+/// Whether the hash is reversed or not depends on the inner type. However you can override it like
+/// this:
```

Comment:
```suggestion
/// Whether the hash is reversed or not when displaying depends on the inner type. However you can
/// override it like this:
```

Reply:
Bizarre that these suggestions are _below_ the other ones on this page since they make suggested changes to code earlier in the file?

Reply:
I believe GH changed recently to put the comments here in the order you wrote them.

Reply:
Cool, I can use "as above" without it getting put in the wrong place - lets see how this goes.